lib/src/workmanager.dart

```diff
@@ -73,8 +75,13 @@ class Workmanager {
   /// A helper function so you only need to implement a [BackgroundTaskHandler]
   static void executeTask(final BackgroundTaskHandler backgroundTask) {
     WidgetsFlutterBinding.ensureInitialized();
-    _backgroundChannel
-        .setMethodCallHandler((call) async => backgroundTask(call.arguments));
+    _backgroundChannel.setMethodCallHandler((call) async {
+      final payload = call.arguments["be.tramckrijte.workmanager.PAYLOAD"];
+      backgroundTask(
+        call.arguments["be.tramckrijte.workmanager.DART_TASK"],
+        payload == null ? null : jsonDecode(payload),
```

Comment:
`payload ?? jsonDecode(payload)`

Reply:
"??" Mean if null. But we need if NOT null

Reply:
`payload == null ? null : jsonDecode(payload)`
if payload is `null` return `null` if it is not return the output of `jsonDecode(payload)`

`payload ?? jsonDecode(payload)`
if payload is null return it if it is not return the output of `jsonDecode(payload)` 

What am I missing?

Reply:
payload ?? jsonDecode(payload)
if payload is null return jsonDecode(payload) if it is not return it

![image](https://user-images.githubusercontent.com/4225279/65031132-62816d00-d949-11e9-9fd0-fbc92649131b.png)

Reply:
right i see now. excuse me need some more sleep

Reply:
It is ok =)